Rasa_Bot/.keras/keras.json

```diff
@@ -0,0 +1,6 @@
+{
+    "floatx": "float32",
+    "epsilon": 1e-07,
+    "backend": "tensorflow",
+    "image_data_format": "channels_last"
+}
```

Comment:
Did you intend to commit this? Is this because in the back rasa uses keras?

Reply:
And do we really need to specify the image_data_format? We don't use image data?

Reply:
If this is all necessary for rasa and you also don't know why that's also fine and you can ignore these comments ;)

Reply:
Honestly I also do not know what it is used for. It appeared when I installed the new rasa version.

Reply:
@svenvanderburg I also just realized, I also need to update the requirements file outside of the Rasa_Bot-folder based on the new Rasa version right? I had not done that yet.

Reply:
OK, good to check if it's necessary to commit it. You can also add it to `.gitignore`.

Reply:
Yes, you need to update the root-folder requirements file!

Reply:
Okay, I have updated the root-folder requirements and added .keras to gitignore

Reply:
Ill merge then once the tests have completed.